modules/ols-embed-knowledge-into-the-llm.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/about/ols-about-openshift-lightspeed.adoc
+
```

Comment:
@syedriko I think it would be helpful if we came up with a name for the tool. Just wanted to check in and see if you were part of any discussions pertaining to naming the tool.